Build fails

```
[  0%] Built target silkworm-buildinfo-git
[  0%] Updating silkworm-buildinfo:
       Project Version:  0.1.0-dev+commit.b709e6f5.dirty (prerelease)
       System Name:      linux
       System Processor: x86_64
       Compiler ID:      gnu
       Compiler Version: 11.2.0
       Build Type:       release
       Git Info:         //b709e6f5ca6fe3b28f195df331898c08f1f281a4 (dirty)
       Git Branch:       master
       Git Origin URL:   https://github.com/torquem-ch/silkworm.git
       Timestamp:        2022-10-20T12:37:23
[  0%] Building C object CMakeFiles/silkworm-buildinfo.dir/silkworm/buildinfo.c.o
[  0%] Linking C static library silkworm/libsilkworm-buildinfo.a
[  0%] Built target silkworm-buildinfo
[  1%] Built target instructions
[  3%] Built target evmone
[  4%] Built target gmplib
[  6%] Built target cborcpp
[  6%] Built target loader
[  6%] Built target tooling
[  7%] Built target mdbx-static
[  7%] Built target mdbx_chk
[  7%] Built target mdbx_copy
[  8%] Built target mdbx_stat
[  8%] Built target mdbx_dump
[  8%] Built target mdbx_load
[  9%] Built target mdbx_drop
[ 39%] Built target torrent-rasterbar
[ 39%] Built target secp256k1
Consolidate compiler generated dependencies of target ff
[ 44%] Built target ff
Consolidate compiler generated dependencies of target silkpre
[ 45%] Built target silkpre
Consolidate compiler generated dependencies of target silkworm_core
[ 45%] Building CXX object core/CMakeFiles/silkworm_core.dir/silkworm/chain/config.cpp.o
In file included from /usr/src/silkworm/core/silkworm/common/util.hpp:28,
                 from /usr/src/silkworm/core/silkworm/chain/config.hpp:29,
                 from /usr/src/silkworm/core/silkworm/chain/config.cpp:17:
/usr/src/silkworm/core/silkworm/common/base.hpp:41:1: error: ‘concept’ does not name a type
   41 | concept UnsignedIntegral = std::unsigned_integral<T> || std::same_as<T, intx::uint128> ||
      | ^~~~~~~
/usr/src/silkworm/core/silkworm/common/base.hpp:41:1: note: ‘concept’ only available with ‘-std=c++20’ or ‘-fconcepts’
In file included from /usr/src/silkworm/core/silkworm/chain/config.cpp:17:
/usr/src/silkworm/core/silkworm/chain/config.hpp:135:17: error: defaulted ‘bool silkworm::operator==(const silkworm::ChainConfig&, const silkworm::ChainConfig&)’ only available with ‘-std=c++20’ or ‘-std=gnu++20’
  135 |     friend bool operator==(const ChainConfig&, const ChainConfig&) = default;
      |                 ^~~~~~~~
In file included from /usr/src/silkworm/core/silkworm/chain/config.cpp:17:
/usr/src/silkworm/core/silkworm/chain/config.hpp:142:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  142 |     .chain_id = 1,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:143:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  143 |     .seal_engine = SealEngineType::kEthash,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:144:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  144 |     .evmc_fork_blocks =
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:156:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  156 |     .dao_block = 1'920'000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:157:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  157 |     .muir_glacier_block = 9'200'000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:158:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  158 |     .arrow_glacier_block = 13'773'000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:159:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  159 |     .gray_glacier_block = 15'050'000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:160:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  160 |     .terminal_total_difficulty = intx::from_string<intx::uint256>("58750000000000000000000"),
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:165:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  165 |     .chain_id = 3,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:166:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  166 |     .seal_engine = SealEngineType::kEthash,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:167:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  167 |     .evmc_fork_blocks =
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:179:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  179 |     .muir_glacier_block = 7'117'117,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:180:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  180 |     .terminal_total_difficulty = 50000000000000000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:185:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  185 |     .chain_id = 4,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:186:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  186 |     .seal_engine = SealEngineType::kClique,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:187:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  187 |     .evmc_fork_blocks =
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:203:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  203 |     .chain_id = 5,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:204:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  204 |     .seal_engine = SealEngineType::kClique,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:205:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  205 |     .evmc_fork_blocks =
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:217:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  217 |     .terminal_total_difficulty = 10790000,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:222:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  222 |     .chain_id = 11155111,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:223:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  223 |     .seal_engine = SealEngineType::kEthash,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:224:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  224 |     .evmc_fork_blocks =
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:237:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  237 |     .muir_glacier_block = 0,
      |     ^
/usr/src/silkworm/core/silkworm/chain/config.hpp:238:5: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
  238 |     .terminal_total_difficulty = 17000000000000000,
      |     ^
cc1plus: all warnings being treated as errors
make[2]: *** [core/CMakeFiles/silkworm_core.dir/build.make:76: core/CMakeFiles/silkworm_core.dir/silkworm/chain/config.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1792: core/CMakeFiles/silkworm_core.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,9 +23,6 @@ endif()
 
 get_directory_property(SILKWORM_HAS_PARENT PARENT_DIRECTORY)
 if(NOT SILKWORM_HAS_PARENT)
-  # Enable language *before* indirectly including GNUInstallDirs (see CableBuildInfo)
-  enable_language(CXX)
-
   include(third_party/evmone/cmake/cable/bootstrap.cmake)
   include(CableBuildType)
   include(CableBuildInfo)

